src/components/Layout/Layout.scss

```diff
@@ -50,10 +54,18 @@ $relative-size: $primary-basis / $secondary-basis;
   margin-top: spacing(loose);
   margin-left: spacing(loose);
 
+  .newDesignLanguage & {
+    margin-top: spacing();
```

Comment:
Seeing a bunch of this spacing stuff is making me think we should have css-tokens for spacing (which IMO we wanted to do anyway), which would mitigate a needing these extra style declarations, but that's a problem for another day :)